app/components/UI/BackupAlert/BackupAlert.styles.ts

```diff
@@ -0,0 +1,74 @@
+import { StyleSheet } from 'react-native';
+import { fontStyles } from '../../../styles/common';
+import Device from '../../../util/device';
+import { Theme } from '../../../util/theme/models';
+
+/**
+ * Style sheet function for ModalConfirmation component.
```

Comment:
update header comment
```suggestion
 * Style sheet function for BackupAlert component.
```

Reply:
Thanks for the review <3

updated!